### DISCUSSION TITLE
Comic book reader

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#151] [Features]

**Comment by AlfHou** [comment]
What is CBZ/CBR?

**Comment by elouanmailly** [comment]
It's a file format for comic books, like epub but for images
https://fr.m.wikipedia.org/wiki/Comic_book_archive

**Reply by carif** [reply]
Technically, an epub can store images and thus comics/mangas as well. In fact, several of my comics are epub-comics and not cbz/cbr files.

**Comment by carif** [comment]
Just found this:

1. Would be cool if such a reader could allow for bookmarks and 
2. to read from left-to-right and right-to-left depending on the comic/manga. Certain metadata formats like the ComicInfo format can store information whether it is a manga or a comic from which one can derive how to scroll through the pages. 
3. Could be nice to change the background color, e.g. when a page doesn't fit the screen to then change the background color to e.g. black or white.

[The Kobo apps ](https://www.kobo.com/gb/en/p/apps) are a great example of a nice and intuitive comic book reader ;)
